haystack/nodes/prompt/prompt_node.py

```diff
@@ -73,7 +73,7 @@ def __init__(
         :param default_prompt_template: The default prompt template to use for the model.
         :param output_variable: The name of the output variable in which you want to store the inference results.
             If not set, PromptNode uses PromptTemplate's output_variable. If PromptTemplate's output_variable is not set, the default name is `results`.
-        :param max_length: The maximum length of the generated text output.
+        :param max_length: The maximum number of characters the generated text output can have.
```

Comment:
Hey @agnieszka-m I don't think is correct. `max_length` refers to the maximum number of tokens that can be generated in the output. Unfortunately, there isn't a definitive way of calculating the length of a token in characters (and it changes depending on what model you use). Here is an [article](https://help.openai.com/en/articles/4936856-what-are-tokens-and-how-to-count-them) from OpenAI that explains this concept.

Reply:
Also I think we should update this in the `PromptModel` docstring too.